lib/kramdown/parser/kramdown/link.rb

```diff
@@ -92,8 +92,9 @@ def parse_link
           emit_warning = !@src[1]
           link_id = normalize_link_id(@src[1] || alt_text)
           if @link_defs.key?(link_id)
-            add_link(el, @link_defs[link_id][0], @link_defs[link_id][1], alt_text,
-                     @link_defs[link_id][2] && @link_defs[link_id][2].options[:ial])
+            link_def_id = @link_defs[link_id]
```

Comment:
That would be a `link_def`, not a `link_def_id`.

Reply:
Agreed. :slightly_smiling_face: